src/main/java/uk/gov/hmcts/reform/sandl.snlevents/model/db/UserTransaction.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.reform.sandl.snlevents.model.db;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.UUID;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.Id;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class UserTransaction {
+
+    @Id
+    @Getter
+    @Setter
+    private UUID id;
+
+    @Getter
+    @Setter
+    @Enumerated(EnumType.STRING)
+    private UserTransactionStatus status;
+
+    @Getter
+    @Setter
+    @Enumerated(EnumType.STRING)
+    private UserTransactionRulesProcessingStatus rulesProcessingStatus;
+
+    @Getter
+    @Setter
+    @JsonIgnore
```

Comment:
It's approved although I'm not sure why there is JsonIgnore here, haven't we agreed to create separate Response classes?